web/pandas/about/team.md

```diff
@@ -52,7 +52,15 @@ The project governance is available in the [project governance page]({{ base_url
 ### {{ workgroup.name }}
 
 <ul>
-    <li><b>Contact:</b> <a href="mailto:{{ workgroup.contact }}">{{ workgroup.contact }}</a></li>
+    <li><b>Contact:</b>
+        <script TYPE="text/javascript">
+            var email='{{ workgroup.contact }}'
+            document.write(
+              '<a href="mailto:' + email + '">'
+              + email + '</a>'
+            )
```

Comment:
Then here you can have something like `$('a.contact').attr('href') = $('a.contact').attr('href').removepreffix('asp.`)`. (sorry for the free style pseudo-code here, but I think you get the idea.

Actually, if we do this, we don't need to create the other javacript dinamically.

The advantage that I see is that what I'm proposing would be resistent if the html document is processes with a regular expression, while your approach is only resistant if it's processed via extracting the `a` tags from the dom.

What do you think?

Reply:
Yep, exactly. Sorry, it was a bit tricky to explain and took some time make my idea understandable. Feel free to implement it the way you think it's simpler and clearer, I was just showing some pseudocode to show better the concept. Also, not sure if `asp.` is the best prefix, also up to you to decide what to use.

Thanks for working on this @Kabiirk

Reply:
No worries :)
Happy to help !